crates/ruff_formatter/src/printer/mod.rs

```diff
@@ -1049,7 +1049,7 @@ struct FitsMeasurer<'a, 'print> {
     bomb: DebugDropBomb,
 }
 
-impl<'a, 'print> FitsMeasurer<'a, 'print> {}
+impl FitsMeasurer<'_, '_> {}
```

Comment:
what's this `impl` block even for?! It's completely empty

Reply:
It's decorative.